eZ/Publish/SPI/Persistence/UserPreference/UserPreference.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Persistence\UserPreference;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+class UserPreference extends ValueObject
+{
+    /**
+     * ID of the user preference.
+     *
+     * @var int
+     */
+    public $id;
+
+    /**
+     * The ID of the user this user preference belongs to.
+     *
+     * @var mixed
```

Comment:
I still believe that `$userId` will be always int, especially due to enforcing of strict types in the implementation :P